src/main/kotlin/org/jmailen/gradle/kotlinter/support/EditorConfigUtils.kt

```diff
@@ -1,8 +1,20 @@
-package org.jmailen.gradle.kotlinter.tasks
+package org.jmailen.gradle.kotlinter.support
 
+import com.pinterest.ktlint.core.api.DefaultEditorConfigProperties
+import com.pinterest.ktlint.core.api.EditorConfigOverride
 import org.gradle.api.file.ProjectLayout
 import java.io.File
 
+internal fun editorConfigOverride(ktLintParams: KtLintParams): EditorConfigOverride {
```

Comment:
The `KtLintEditorConfigOverrides.kt` was initially reported by `filename` check, so I realized we do already have a _container_ for EditorConfig-related utilities. I moved the utiliy from `.tasks` to `.support` package, as that seemed to be a proper place for such utilities 👀